tests/vcs/git/GitCommitBearTest.py

```diff
@@ -100,6 +100,29 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_github_pull_request_temporary_merge_commit(self):
+        self.run_git_command('remote', 'add', 'upstream',
+                             'git://github.com/kriti21/coala-bears.git')
+        run_shell_command(
+            'git fetch upstream pull/3/merge:good_test')
+        run_shell_command('git checkout good_test')
+        self.assertEqual(self.run_uut(), [])
+
+        # If parent of this merge commit is wrong, message is
+        # displayed accordingly.
+        self.run_git_command('remote', 'add', 'upstream',
+                             'git://github.com/kriti21/coala-bears.git')
+        run_shell_command(
+            'git fetch upstream pull/2/merge:bad_test')
+        run_shell_command('git checkout bad_test')
+        self.assertEqual(self.run_uut(),
+                         ['Shortlog of the HEAD commit contains 70 '
+                          'character(s). This is 20 character(s) longer '
+                          'than the limit (70 > 50).'])
```

Comment:
AppVeyor is failing here, because the shortlog of the HEAD commit there contains 72 chars.
Investigate why that is happening.

Maybe related to newline characters having a different length on Windows.

Reply:
Locally it runs fine. I changed it to 70 in my last build but I think that fails circleci and travis

Reply:
I am using Ubuntu Linux

Reply:
You need to work out what the string is on Windows.  Use `repr` to get a better look at the string.

Do it on another branch, with *only* AppVeyor CI running on your fork.